config/kubernetes/sig-testing/teams.yaml

```diff
@@ -37,3 +37,14 @@ teams:
     - cjwagner
     - mkumatag
     privacy: closed
+  sig-testing-bash-reviews:
```

Comment:
If you drop sig-testing from the name, then I'm fine with it, though I still think this is a volunteer effort at best and may not improve the situation.  All of these folks are pretty oversubscribed as it is